pkg/ui/chooser/embed/index.html

```diff
@@ -16,11 +16,21 @@
             grid-template-columns: 1fr 1fr;
             column-gap: 2rem;
         }
+
+        .logo {
+            display: block;
+            width: 250px;
+            height: auto;
+        }
     </style>
+
+    <link rel="icon" type="image/x-icon" href="../../../../website/static/favicon.ico">
```

Comment:
those files would actually need to be copied over so that they can be embedded in the Go binary

Reply:
Ah yes! that was an oversight

Reply:
Please, can you elaborate more on what you mean by copy over

Reply:
I mean that we shouldn't reference files in the repo, but instead reference files that will be embedded in the Go binary, like `simple.min.css`

Reply:
Like the aim and mlflow ui src/src, does the chooser ui already have a directory for modifications. I presently can’t find an existing one in this repo (fasttrackml)

Reply:
yeah the other UIs require "building" and so that is why it's then converted — the chooser UI is (so far) just plain HTML/CSS/JS, so things have to be dropped in the right place instead